src/client/Client.cc

```diff
@@ -6019,7 +6019,8 @@ int Client::get_or_create(Inode *dir, const char* name,
 {
   // lookup
   ldout(cct, 20) << "get_or_create " << *dir << " name " << name << dendl;
-  dir->open_dir();
+  Dir *dircheck = dir->open_dir();
+  assert(dircheck);
```

Comment:
I don't understand. Inode::open_dir() is conditioned on this and won't return NULL.